docs/UserGuide.adoc

```diff
@@ -209,79 +172,104 @@ Format: `find n\[name]`
 
 Example:
 
-* `find n\Jane doe` +
-Finds a saved contact named Jane Doe.
+* Suppose you want to find Jane doe within NOVA,
++
+`find n\Jane doe`
++
+NOVA will find and list saved contacts named Jane Doe.
+
+* Suppose you want to find Jane within NOVA,
```

Comment:
Can spice it up a little:

* Or you can find Jane Doe with just Jane,

   `find n\Jane`
   ...